src/main/java/uk/gov/companieshouse/api/accounts/validation/IntangibleAssetsValidator.java

```diff
@@ -141,6 +126,64 @@ private void validateSubResourceTotals(IntangibleAssets intangibleAssets, Errors
     private void validateSubResourceTotal(IntangibleAssetsResource intangibleAssetsResource, Errors errors, boolean isMultipleYearFiler, IntangibleSubResource subResource) {
 
         validateCosts(intangibleAssetsResource, errors, subResource);
+        validateAmortisation(intangibleAssetsResource, errors, subResource);
+    }
+
+    private void validateAmortisation(IntangibleAssetsResource intangibleAssetsResource, Errors errors, IntangibleSubResource subResource) {
+
+        Long chargeForYear = getChargeForYear(intangibleAssetsResource);
+        Long onDisposal = getOnDisposals(intangibleAssetsResource);
+        Long otherAdjustments = getOtherAdjustments(intangibleAssetsResource);
+        Long atPeriodStart = getAmortisationAtPeriodStart(intangibleAssetsResource);
+
+        Long calculatedAtPeriodEnd = atPeriodStart + chargeForYear + onDisposal + otherAdjustments;
```

Comment:
I believe it should be:

Long calculatedAtPeriodEnd = atPeriodStart + chargeForYear - onDisposal + otherAdjustments;